### ISSUE TITLE
An option to move/remove a card by clicking/tapping instead of dragging

### ISSUE THREAD

**Issue by jan-kleks** [KyleU/solitaire.gg#7]

**Comment by KyleU** [comment]
You can double-click/tap a card to automatically move it - is this what you meant?

**Comment by jan-kleks** [comment]
Exactly. You can add this info to [the "tips" section](https://github.com/KyleU/solitaire.gg/issues/3#issuecomment-291004490) too.

**Comment by jan-kleks** [comment]
Okay, I've found two problems. First of all, you can only move only one card using this method, it would be nice to be able to move two or even more cards (when the rules allow this, of course).

Secondly, if two or moves are available for a given card, how does your app chooses which move would be best? Maybe it should show all the available options to the player in such a case. But wouldn't that be cheating? An alternative to all this would be the "point and select a card/cards and then tap" option.

I simply want to avoid dragging on mobile devices.

**Comment by jan-kleks** [comment]
You can also consider the point-and-click option for the mouse users.

**Comment by jan-kleks** [comment]
After considering this even further I really think you should change the functionality of double-click/tap on a card. Maybe it should only move cards to foundations (if there's free space). Point-and-click would be really a good alternative for those who want to avoid dragging, and there won't be issues with automatically choosing the right spot for the card (https://github.com/KyleU/solitaire.gg/issues/7#issuecomment-291329484).
